Cargo.toml

```diff
@@ -27,6 +27,7 @@ lto = "thin"
 [features]
 embed-pictrs = ["pict-rs"]
 console = ["console-subscriber", "opentelemetry", "opentelemetry-otlp", "tracing-opentelemetry", "reqwest-tracing/opentelemetry_0_16"]
+prom = ["prometheus", "actix-web-prom"]
```

Comment:
Call it prometheus or even prometheus-metrics, otherwise its confusing. Same for prom.rs

Reply:
Yeah I tried calling it `prometheus` originally, but apparently you can't have a feature name and a dependent package with the same name. Updated it to `prometheus-metrics` now.